### PR TITLE
[Avalon StarGO] Enabling/disabling focuser

### PR DESCRIPTION
**Pull request by sterne-jaeger**
Querying the focuser status without a focuser connected leads to problems with StarGO when running several hours. It might lead that the StarGO device no longer responds and requires a reset.

With this version, the focuser can be activated and deactivated avoiding unnecessary (and potentially troublesome) querying to the AUX device.

### CODE REVIEW

**Comment by knro** [comment]
Great! Maybe also change debian/indi-avalon/changelog  as well to sync the package version?

File: debian/indi-avalon/changelog [hunk 1]
```diff
@@ -1,5 +1,46 @@
-indi-avalon (1.0) bionic; urgency=medium
+indi-avalon (1.3) bionic; urgency=medium
```
Comment (knro) [inline debian/indi-avalon/changelog]: This is invalid format... just duplicate my first entry tha

Comment (knro) [inline debian/indi-avalon/changelog]: This is an invalid changelog .. the debian one is used to build packages and have very specific format. Please duplicate my first entry and modify it. or use **dch** to append a new changelog. We have a problem just yesterday with indi-starbook failing to build for a couple of weeks and it was traced to a _single space_ that wasn't added!

**Comment by sterne-jaeger** [comment]
Hope it's correct now. How can I test it?

**Comment by knro** [comment]
This is fine now, we'll know soon if there are issues with the debian builds!

**Pull request merged**